### PR TITLE
Reorganize blueprints

### LINKED ISSUE #129: Reorganize functions in blueprints
Better associate functions with their purpose - probably involve moving some things (search) to different blueprints/files

### PR DESCRIPTION
**Pull request by msoucy**
Make the endpoints and URLs make a bit more sense

Closes #129

### CODE REVIEW

**Review by jeffaburke — APPROVED** [review]
I like the rearrangement, though I'm gonna let Ben double check this one

File: signinapp/dbadmin.py [hunk 1]
```diff
@@ -47,14 +47,14 @@ def init_app(app):
...
+        AuthModelView(Badge, db.session, endpoint="admin_badge"),
+        AuthModelView(Event, db.session, endpoint="admin_event"),
         AuthModelView(EventType, db.session, endpoint="eventtype"),
         AuthModelView(Guardian, db.session, endpoint="guardian"),
         AuthModelView(Role, db.session, endpoint="role"),
         AuthModelView(Student, db.session, endpoint="student"),
         AuthModelView(Subteam, db.session, endpoint="subteam"),
-        AuthModelView(User, db.session, endpoint="adminuser"),
+        AuthModelView(User, db.session, endpoint="admin_user"),
         AuthModelView(Stamps, db.session, endpoint="stamps"),
```
Comment (bot190) [inline signinapp/dbadmin.py]: Aren't all of these at /dbadmin/* do they need to be re-named?

File: signinapp/active.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        delete(Active)
+        .where(Active.event.has(is_active=False))
+        .execution_options(synchronize_session="fetch"),
+    )
+    db.session.commit()
+    flash("Deleted all expired stamps")
+    return redirect(url_for("mentor.active"))
+
+
+@bp.route("/admin/active/delete_all")
```
Comment (bot190) [inline signinapp/active.py]:
```suggestion
@bp.route("/delete_all")
```

File: signinapp/dbadmin.py (hunk 1 above)
Comment (bot190) [reply signinapp/dbadmin.py]: And if they aren't, then I think we should just update all of these with the `admin_` prefix.

**Review by bot190 — CHANGES_REQUESTED** [review]

File: signinapp/dbadmin.py (hunk 1 above)
Comment (msoucy) [reply signinapp/dbadmin.py]: These are the endpoint, which is different from the url prefix. I'm updating the endpoint names though.

**Review by bot190 — APPROVED** [review]
Changes look good, ready to merge once it's rebased.

**Pull request merged**